client/src/node/extension.ts

```diff
@@ -48,6 +48,7 @@ import { NotebookSerializer } from "../components/notebook/Serializer";
 import { ConnectionType } from "../components/profile";
 import { SasTaskProvider } from "../components/tasks/SasTaskProvider";
 import { SAS_TASK_TYPE } from "../components/tasks/SasTasks";
+import { getSelectedRegions } from "../utils/utils";
```

Comment:
@jingjiajie same here, it currently fails the build

Reply:
done.